.vscode/launch.json

```diff
@@ -13,6 +13,13 @@
         "${workspaceFolder}/tools/**/*.js"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "(made) Run current file",
```

Comment:
Is there a more clear name for this command that you could give it? I'm not sure what "(made) Run current file" is supposed to do/mean.

Reply:
I made it so I could distinguish between that and another setting I can't seem to find, and forgot to turn it into a PR friendly name lol. Great catch!